lib/widgets/message_list.dart

```diff
@@ -968,25 +968,32 @@ class MessageItem extends StatelessWidget {
     this.trailingWhitespace,
   });
 
-  final MessageListMessageItem item;
+  final MessageListMessageBaseItem item;
```

Comment:
nit:

    msglist [nfc]: Handle MessageListMessageBaseItem with MessageItem

Can expand slightly to clarify:

    msglist [nfc]: Handle MessageListMessageBaseItem with MessageItem widget

Otherwise, when looking at the commit message before rereading the code, these two classes sound like the same kind of thing, and then their relationship is confusing.

Reply:
Or I think better yet:

    msglist [nfc]: Let MessageItem widget take a MessageListMessageBaseItem

That basically specifies that we're talking about a field on the widget, vs. the wide range of other things that "handle" could mean.